gp-okta.py

```diff
@@ -446,6 +446,9 @@ def main():
 	conf = load_conf(sys.argv[1])
 	
 	s = requests.Session()
+        if conf.get('cert'):
```

Comment:
Please check your tabs vs. spaces here and in your other commit. I had to fix a few places that errored out with:
```
  File "./gp-okta.py", line 390
    if state_token is not None:
                              ^
TabError: inconsistent use of tabs and spaces in indentation
```

Reply:
Oh, you are right. I tested with python2.7 -- that one did not complain, whereas python3.x does.

Fixed.